spec/requests/citizens/own_home_spec.rb

```diff
@@ -21,35 +21,24 @@
       let(:params) { { 'legal_aid_application' => { 'own_home' => 'owned_outright' } } }
 
       it 'returns http_success' do
-        expect(response).to have_http_status(:ok)
+        expect(response).to have_http_status(:found)
```

Comment:
`:found` matches 302, which is a redirect. It's usually a good idea to specify where the redirect points at in the spec as there are a number of reasons why a redirect may occur (authentication failure for example). So just detecting a redirect isn't particularly useful.

Also - if the response is a redirect, the `it` text `'returns http_success'` is no longer valid.

So this should probably be replaced with something like:
```ruby
it 'redirects to ...<the destination>' do	          
    expect(response).to redirect_to(<destination_path>)
end
```
Replaces the `<..>` content with the appropriate entries.